### PR TITLE
add bootstrap-daterangepicker@1.3.22 with git autoupdate

### PR DESCRIPTION
**Pull request by LeaYeh**

### CODE REVIEW

File: ajax/libs/daterangepicker/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "node": "0.10.32",
+    "nmp": "1.4.28"
+  },
+  "dependencies": {
+    "jquery": "^1.10",
+    "moment": "^2.9.0"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com:ghiden/angucomplete-alt.git",
```
Comment (PeterDaveHello) [inline ajax/libs/daterangepicker/package.json]: Wrong syntax again as in https://github.com/cdnjs/cdnjs/commit/b3c61c32b3682b3df5781b0032f9796ec7c6d38e ...

Comment (PeterDaveHello) [inline ajax/libs/daterangepicker/package.json]: And what's the reason why that you added a totally unrelated repo as its source here?

**Comment by PeterDaveHello** [comment]
Please manually minify the css files.

**Comment by LeaYeh** [comment]
done.

**Comment by PeterDaveHello** [comment]
No you are not, still wrong syntax and wrong repo in git auto-update config.

**Comment by LeaYeh** [comment]
fixed url syntax and repo ref.

File: ajax/libs/daterangepicker/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  "dependencies": {
+    "jquery": "^1.10",
+    "moment": "^2.9.0"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/dangrossman/bootstrap-daterangepicker.git",
+    "basePath": "",
+    "files": [
+      "daterangepicker.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/daterangepicker/package.json]: There is no `daterangepicker.min.js` in its repo...

File: ajax/libs/daterangepicker/package.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      "daterangepicker-bs2.css",
+      "daterangepicker-bs3.css",
+      "daterangepicker-bs3.min.css",
+      "daterangepicker-bs2.min.css",
+      "daterangepicker.js",
+      "daterangepicker.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/daterangepicker/package.json]: Duplicated.

**Comment by PeterDaveHello** [comment]
BTW, I think its name should be `bootstrap-daterangepicker`
https://github.com/dangrossman/bootstrap-daterangepicker/blob/master/package.json

File: ajax/libs/daterangepicker/package.json (hunk 2 above)
Comment (LeaYeh) [inline ajax/libs/daterangepicker/package.json]: 我是手動轉的
不然他本身沒有

File: ajax/libs/daterangepicker/package.json (hunk 3 above)
Comment (LeaYeh) [inline ajax/libs/daterangepicker/package.json]: 你的意思是有 min 就不用原檔了?

Comment (PeterDaveHello) [inline ajax/libs/daterangepicker/package.json]: line 35 ... line 41 ...

**Comment by LeaYeh** [comment]
plz try again ><"

**Comment by PeterDaveHello** [comment]
You didn't update the commit message.

**Comment by LeaYeh** [comment]
what do you mean about  'update' the commit message?

**Comment by PeterDaveHello** [comment]
Still `daterangepicker`?

**Comment by LeaYeh** [comment]
is it correct?

**Comment by PeterDaveHello** [comment]
Should rebase on our latest master branch.

**Comment by LeaYeh** [comment]
我不確定這樣對不對....
他把一堆你這五天來的 commit 都抓下來了...

**Comment by LeaYeh** [comment]
OK.

**Pull request merged**